APIAssignment1.py

```diff
@@ -0,0 +1,97 @@
+import csv
+import json
+
+import requests
+
+import A1constants
+
+
+# helper function to write to csv from list, all lines at once
```

Comment:
Excellent! it's a good habit to write documentation and spec for the code you write.
Perhaps also look at how to write class/ function level comments? you should be able to find resource online